### PR TITLE
Feature/connector customizer

### PR DESCRIPTION
**Pull request by philip-ellis-sp**

### CODE REVIEW

File: products/idn/docs/identity-now/saas-connectivity/connector-commands/source-data-read.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+keywords: ['connectivity', 'connectors', 'Source Data Read']
+description: Create account on the source.
+slug: /docs/saas-connectivity/commands/source-data-read
+tags: ['Connectivity', 'Connector Command']
+---
+
+| Input/Output |            Data Type        |
+| :----------- | :-------------------------: |
+| Input        | StdSourceDataDiscoverInput  |
+| Output       | StdSourceDataDiscoverOutput |
```
Comment (bryant-wong-sp) [inline products/idn/docs/identity-now/saas-connectivity/connector-commands/source-data-read.md:15]: did you mean sourcedataread input/output here?

Comment (philip-ellis-sp) [reply products/idn/docs/identity-now/saas-connectivity/connector-commands/source-data-read.md]: Yes, just pushed a fix for this, thanks for reading!

**Comment by jordan-violet-sp** [comment]
Just wanted to say that I appreciate your use of the Mermaid diagram :)

**Review by james-haytko-sp — APPROVED** [review]
Looks good! I made some edits.

**Pull request merged**